### DISCUSSION TITLE
[FEATURE-REQUEST] Posibilitar reset identifiers con autocompletados como padre

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#240] [3. Ideas & Feature Requests] [answered]
Buenas,

En una implementación necesito hacer reset identifiers con una question que se autocompleta (exec.validate) como padre y no se ejecuta el resetIdentifiers. Esto sí funciona cuando la question padre es completada por el usuario (modificada a mano). 
Específicamente es utilizar una quesiton de tipo número que se autocompleta para resetear una question de tipo listquestion que tiene un (1) elemento cada vez que el valor autocompletado se modifica.

¡Gracias!

**Comment by guillevalin** [comment, marked as answer]
El problema es que el exec.Validate está escribiendo en un campo como efecto de otro campo, realizar acciones anidadas podría tener el riesgo de loops infinitos.

Podrías asociar el otro campo al mismo campo que se usa para calcular el campo de solo lectura para resetIdentifiers y resetear ambos cuando se modifique este.
